dnnsoftware/Dnn.Platform
# Fix resource manager permissions save issues

Fixes https://github.com/dnnsoftware/Dnn.Platform/issues/6045


## Summary
Fix resource manager permissions save issues

- set default PortalID in vw_FolderPermissions to -1
- include other selected permissions when 'Write to Folder' permisison is set
- update javascript to fix allowAccess==0 permissions issues

### Release notes
Resolved an issue where the permissions in the resource manager were not being saved and fixed some of the auto adjust enable and disable permission functions.

### Testing steps
Re-run the steps in https://github.com/dnnsoftware/Dnn.Platform/issues/6045 and check that the issues are solved
